trace/trace.go

```diff
@@ -110,6 +111,9 @@ const (
 	SpanKindClient
 )
 
+// DefaultBufferLimit is the default value for trace.StartOptions.BufferLimit.
+const DefaultBufferLimit = 1000
```

Comment:
The spec have different limits for each entities. Attributes: 32, Message events: 128, ...

https://github.com/census-instrumentation/opencensus-specs/blob/master/trace/TraceConfig.md